src/inspector/types/texttypes.h

```diff
@@ -55,52 +55,65 @@ class TextTypes
         FRAME_TYPE_SQUARE,
         FRAME_TYPE_CIRCLE
     };
-
+    // must match Ms::Tid
     enum class TextType {
         TEXT_TYPE_DEFAULT,
         TEXT_TYPE_TITLE,
         TEXT_TYPE_SUBTITLE,
         TEXT_TYPE_COMPOSER,
         TEXT_TYPE_POET,
-        TEXT_TYPE_LYRICS_ODD,
-        TEXT_TYPE_LYRICS_EVEN,
-        TEXT_TYPE_FINGERING,
-        TEXT_TYPE_LH_GUITAR_FINGERING,
-        TEXT_TYPE_RH_GUITAR_FINGERING,
-        TEXT_TYPE_STRING_NUMBER,
+        TEXT_TYPE_TRANSLATOR,
```

Comment:
technically this is to fix a different issue (https://github.com/musescore/MuseScore/issues/8654), but hard to test current issue without fixing this too